src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -1089,21 +1089,31 @@ export interface TIX {
   getSymbols(): string[];
 
   /**
-   * Returns the price of a stock
+   * Returns the price of a stock.
    *
    * @remarks
    * RAM cost: 2 GB
-   * The stock’s price is the average of its bid and ask price.
+   *
+   * The stock’s price is the average of its bid and ask prices. This function requires
+   * that you have the following:
+   *
+   * 1. WSE Account
+   *
+   * 1. TIX API Access
```

Comment:
> Both are marked as requirement number 1

Markdown is very lenient about how enumerated lists are written.

**Example A:**
The following
```
1. First

2. Second

3. Third
```
would be rendered as


1. First

2. Second

3. Third

This example makes sense even when read without being rendered on GitHub or VSC preview.

**Example B:**
The following
```
1. First

3. Second

4. Third
```
is rendered as

1. First

3. Second

4. Third

This example is confusing when reading the `.md` file (not rendered).  Markdown doesn't care.  The result would still be the same as **Example A**.  The very first number (in this case `1.`) is taken as starting the enumeration.  The confusion compounds when you expand the list to, say:
```
1. First

3. Second

4. Third

7. Seventh

10. Eight
```

which is rendered as:

1. First

3. Second

4. Third

7. Seventh

10. Eight

For me, it's confusing reading a `.md` file with such an enumerated list.  It takes some effort and frustration to maintain and expand the above list.  If in the future I need to add 3 more list items between `4. Third` and `7. Seventh`, I might do this:
```
1. First

3. Second

4. Third

5. Fourth

6. Fifth

6. Sixth

7. Seventh

10. Eight
```
Or put in some effort to re-organize everything in the following human readable form:
```
1. First

2. Second

3. Third

4. Fourth

5. Fifth

6. Sixth

7. Seventh

8. Eight
```
Now suppose I have this human readable list.  This does not prevent some future maintainer from inadvertently doing something silly like so:
```
1. First

2. Second

3. Third

4. Fourth

4. Ninth

5. Fifth

6. Sixth

7. Seventh

8. Eight

11. Twelfth
```
which renders fine as

1. First

2. Second

3. Third

4. Fourth

4. Ninth

5. Fifth

6. Sixth

7. Seventh

8. Eight

11. Twelfth

But it's part of a reviewer's job to catch such mistakes in the `.md` file, no?  Yes, it is.  However, that would put some unnecessary strain on the reviewer to debug why `4. Ninth` is rendered as `5. Ninth`.

It's the job of an IDE like VSC to help developers correctly maintain an enumerated list in Markdown.  I'm very new to VSC and couldn't get it to correctly re-order an enumerated list embedded in JavaScript documentation.  Perhaps VSC fails because it doesn't know that the documentation is intended to be extracted by API Extractor and converted to Markdown?  VSC does its job perfectly well when I edit a `.md` file.

**Example C:**
The following
```
1. First

1. Second

1. Third
```
is rendered as

1. First

1. Second

1. Third

Markdown is happy with this way of writing an enumerated list.  This has the same effect as an HTML ordered list:
```
<ol>
  <li>First</li>
  <li>Second</li>
  <li>Third</li>
</ol> 
```
This reminds me of how Java developers used to write ordered lists in their Java doc: by embedding HTML ordered lists in their documentation.  For me, the way of writing an enumerated list in Markdown (as in **Example C**), and embedded in JavaScript documentation, requires minimal amount of effort to maintain.  To add another ordered item between 2 existing items, I would use the template `1. anotherItem`.